dependent/jospt.csl

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>JOSPT (Journal of Orthopaedic and Sports Physical Therapy)</title>
```

Comment:
Let's just go with the full title here:

`</title>Journal of Orthopaedic & Sports Physical Therapy</title>` (also notice the ampersand)

and "journal-of-orthopaedic-and-sports-physical-therapy.csl"

(see also http://www.ncbi.nlm.nih.gov/nlmcatalog/7908150)